parachain-template/runtime/src/xcm_config.rs

```diff
@@ -180,7 +180,7 @@ impl xcm_executor::Config for XcmConfig {
 	type AssetTrap = PolkadotXcm;
 	type AssetClaims = PolkadotXcm;
 	type SubscriptionService = PolkadotXcm;
-	type PalletInstancesInfo = ();
+	type PalletInstancesInfo = super::AllPalletsWithSystem;
```

Comment:
Import this at the very top of the file instead:

```
use super::AllPalletsWithSystem;
```

Reply:
fixed